### DISCUSSION TITLE
My Account has been Flagged, I can't do anything, and no one is answering for OVER 2 weeks.

### DISCUSSION THREAD

**Discussion by dBish6** [community/community#108109] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

It's been over 2 weeks now without a reply to my support ticket at https://support.github.com/ticket/personal/0/2568501. I opened a ticket about my account being flagged because there was some 'unusual' activity on my account. Nothing happened, no repos got changed, nothing happened, but I had to change my password and now I can't do anything, like pushing, pulling, or whatever because I can't sign into git. This is really bad for me, this happened at the worst possible time because of a project that I had due and now I can't push my new Portfolio also and deploy it, this is really pissing me off. So, I am writing this hoping someone will see this and not take another 2 weeks. I wonder if it was an error on GitHub's side because nothing even happened, I don't know, I really need help as soon as possible. Thanks.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @dBish6, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
